internal/mode/static/nginx/runtime/manager_test.go

```diff
@@ -7,26 +7,137 @@ import (
 	"testing"
 	"time"
 
-	"github.com/go-logr/logr"
 	ngxclient "github.com/nginxinc/nginx-plus-go-client/client"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/nginx/runtime"
+	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/nginx/runtime/runtimefakes"
 )
 
 var _ = Describe("NGINX Runtime Manager", func() {
 	It("returns whether or not we're using NGINX Plus", func() {
-		mgr := NewManagerImpl(nil, nil, logr.New(GinkgoLogr.GetSink()))
+		mgr := runtime.NewManagerImpl(nil, nil, zap.New(), nil, nil)
 		Expect(mgr.IsPlus()).To(BeFalse())
 
-		mgr = NewManagerImpl(&ngxclient.NginxClient{}, nil, logr.New(GinkgoLogr.GetSink()))
+		mgr = runtime.NewManagerImpl(&ngxclient.NginxClient{}, nil, zap.New(), nil, nil)
 		Expect(mgr.IsPlus()).To(BeTrue())
 	})
+
+	var (
+		manager         runtime.Manager
+		upstreamServers []ngxclient.UpstreamServer
+		upstreamServer  ngxclient.UpstreamServer
+		ngxPlusClient   *runtimefakes.FakeNginxPlusClient
+		process         *runtimefakes.FakeProcessHandler
+
+		metrics      *runtimefakes.FakeMetricsCollector
+		verifyClient *runtimefakes.FakeVerifyClient
+	)
+
+	BeforeEach(func() {
+		upstreamServer = ngxclient.UpstreamServer{}
+
+		upstreamServers = []ngxclient.UpstreamServer{
+			upstreamServer,
+		}
+
```

Comment:
```suggestion
		upstreamServers = []ngxclient.UpstreamServer{
			ngxclient.UpstreamServer{},
		}
```
Then you can delete the `upstreamServer` variable declaration above since it isn't used anymore.